lib/active_resource/detailed_log_subscriber.rb

```diff
@@ -20,7 +20,7 @@ def log_request_response_details(event)
       request_body = data.first
 
       info("Request:\n#{request_body}") if request_body
-      info("Headers: #{headers.inspect}")
+      info("Headers: #{headers.except("X-Shopify-Access-Token").inspect}")
       info("Response:\n#{event.payload[:response].body}")
```

Comment:
I think it might be better to 'blank' the token rather than remove, otherwise it may cause confusion when debugging if it appears the header is missing.

Reply:
That makes sense, instead of a blank would it be clearer to put "..." or something like that?

Reply:
Yeah, or maybe `[FILTERED]` to match what Rails does with `filter_parameters`:

https://blog.appsignal.com/2013/04/18/filter-sensitive-data-from-params.html